spirv_msl.hpp

```diff
@@ -349,6 +423,14 @@ class CompilerMSL : public CompilerGLSL
 	// sampler's binding is returned instead. For any other resource type, -1 is returned.
 	uint32_t get_automatic_msl_resource_binding_secondary(uint32_t id) const;
 
+	// Same as get_automatic_msl_resource_binding, but should only be used for combined image samplers for multiplanar images,
+	// in which case the second plane's binding is returned instead. For any other resource type, -1 is returned.
+	uint32_t get_automatic_msl_resource_binding_tertiary(uint32_t id) const;
```

Comment:
Need to update `get_default_extended_decoration` to get this default -1 behavior.